server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/cucumber/UserStepdefs.java

```diff
@@ -145,6 +145,22 @@ public void shareMailbox(String owner, String mailbox, String shareTo) throws Th
         mainStepdefs.aclProbe.addRights(mailboxPath, shareTo, rights);
     }
 
+    @Given("^\"([^\"]*)\" shares its mailbox with write right \"([^\"]*)\" with \"([^\"]*)\"$")
+    public void shareMailboxWithWriteRight(String owner, String mailbox, String shareTo) throws Throwable {
```

Comment:
Rebase on https://github.com/linagora/james-project/pull/1053 and use 

```
 public void shareMailbox(String owner, String mailbox, String shareTo, String rights) throws Throwable {
```

instead...